pyproject.toml

```diff
@@ -33,12 +33,12 @@ pyrsistent = "0.16.1"
 uritemplate = "^3.0.1"
 python-dateutil = "^2.8.2"
 pytz = "^2021.1"
+requests = "^2.27.1"
 
 [tool.poetry.dev-dependencies]
 pyfakefs = "3.7"
 parameterized = "^0.7.4"
-nose = "^1.3.7"
-nose-timer = "^1.0.0"
+pytest = {version = "^6.2.5", python = ">= 3.6"}
```

Comment:
Since we don't support versions less then 3.7, can drop the `python = ">= 3.6"` specifier here